build/yamls/flow-aggregator.yml

```diff
@@ -0,0 +1,80 @@
+apiVersion: v1
```

Comment:
What is difference between this file and build/yamls/flow-aggregator/flow-aggregator.yml

Reply:
Thanks, build/yamls/flow-aggregator/flow-aggregator.yml has been deleted since we use kustomize to generate build/yamls/flow-aggregator.

Reply:
As there is separate folder for flow-aggregator, why don't we generate at build/yamls/flow-aggregator/flow-aggregator.yml and not at build/yamls/flow-aggregator.yml?

Reply:
Yeah this location doesn't matter too much I think, I was following the style in build/yamls/octant/ or windows/, the yaml files they generate are under build/yamls/ but not their own subdirectory.

Reply:
Following others' convention sounds fine to me.